### PR TITLE
WEB CU 8687300TX Read from Master Member Tracking

### PR DESCRIPTION
**Pull request by nlchung**
Google Sheets setup, /getuserinfo command

### CODE REVIEW

**Comment by sherwinchiu** [comment]
Task linked: [CU-8687300tx Read from Master Member Tracking](https://app.clickup.com/t/8687300tx)

File: commands/utility/get_user_info.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const { SlashCommandBuilder, PermissionFlagsBits } = require('discord.js');
+/*  ----------------------
+Google Sheets Setup
+-------------------------- */
+const { spreadsheetId } = require('../../config.json');
+const { google } = require('googleapis');
+
+const auth = new google.auth.GoogleAuth({
```
Comment (sherwinchiu) [inline commands/utility/get_user_info.js:8]: Does this require auth on every call for get_user_info command? or is it just executed once (mb idk how it works)

**Review by YutongZhuu — APPROVED** [review]
The new command is fully functional.

File: commands/utility/get_user_info.js (hunk 1 above)
Comment (YutongZhuu) [reply commands/utility/get_user_info.js:8]: I think it only runs when we run the deploy command file, but i am not too sure. (new to discord.js too)

Comment (nlchung) [reply commands/utility/get_user_info.js:8]: not totally sure -- will look into it n fix if needed

**Pull request merged**